pyproject.toml

```diff
@@ -0,0 +1,26 @@
+[tool.ruff]
+exclude = ["sparse/_version.py"]
```

Comment:
Is this line needed? I think it's still coming from another project

Reply:
Removed. Thanks.